app/assets/javascripts/protocols/steps.js.erb

```diff
@@ -545,11 +545,29 @@
     var $tinyMCEInput = TinyMCE.getContent();
     var descriptionValid = textValidator(ev, $descrTextarea, 0,
       <%= Constants::TEXT_MAX_LENGTH %>, false, $tinyMCEInput);
-
+    var tableNamesValidArray = [];
+    // iterate over table titles and validate their length
+    $form.find(".table_name").each(function(index , tableName) {
+      if (tableName.value.length > 0) {
+        tableNamesValidArray[index] = textValidator(ev, tableName, 1,
+          <%= Constants::NAME_MAX_LENGTH %> );
```

Comment:
constants are still used here, because this will get merge before js.erb issue #1288